code-gen/parse-csl/src/Csl/Parse.hs

```diff
@@ -58,11 +58,27 @@ parseMethods str = toMethod <$> funBody str
 
     rmStaticPrefix x = x { fun'name = drop 7 $ fun'name x }
 
+toEnumParts :: String -> [String]
+toEnumParts = tail . splitOn enumPrefix
+
+enums :: String -> [CslEnum]
+enums = mapMaybe parseEnum . toEnumParts
+
+parseEnum :: String -> Maybe CslEnum
+parseEnum str = do 
+  (name, rest1) <- split2 ": {|" str
+  (cases, _) <- split2 "|}" rest1 
+  pure $ CslEnum (trim name) (parseEnumCases cases)  
+
+parseEnumCases :: String -> [String]
+parseEnumCases = map (fst . breakOn ":") . tail . splitOn "+"
+
 ----------------------------------------------------------------------------
 -- const
 
 classPrefix = "declare export class "
 funPrefix = "declare export function "
+enumPrefix = "declare export var "
 
 -- | Taken from github repo  <github:emurgo:cardano-serialization-lib/rust/pkg/cardano_serialization_lib.js.flow>
```

Comment:
Here, it would be very nice to specify the revision/version where it was taken cc @marcusbfs

Reply:
Addressed in https://github.com/mlabs-haskell/purescript-cardano-serialization-lib/pull/4/commits/a6206eede1340847629266966741ea67ca1df7b7